components/brave_ads/core/internal/account/confirmations/reward/reward_info.cc

```diff
@@ -0,0 +1,58 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_ads/core/internal/account/confirmations/reward/reward_info.h"
+
+#include <cmath>
+#include <tuple>
+
+namespace brave_ads {
+
+RewardInfo::RewardInfo() = default;
+
+RewardInfo::RewardInfo(const RewardInfo& other) {
+  *this = other;
+}
+
+RewardInfo& RewardInfo::operator=(const RewardInfo& other) {
+  if (this != &other) {
```

Comment:
can we just make this `= default`?

Reply:
Ah yes, we required due to Cloning, before the refactor. Fixed. Thanks